kolibri/core/analytics/test/test_utils.py

```diff
@@ -273,6 +297,7 @@ def test_extract_facility_statistics(self):
         }
 
         assert actual == expected
+        self.assertNotIn("sh", actual)
```

Comment:
I guess it's implied by the equivalence assertion, but good to have it be explicit, yeah!